### PR TITLE
Updates to readme to reflect using Go

### PR DESCRIPTION
**Pull request by CorneliusIV**
This connects to #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,35 @@
...
 
-## Installation
+## Getting Started
+Install [Go](http://golang.org/doc/install.html) and set up your [GOPATH](https://golang.org/doc/code.html#GOPATH)
 
-1. Since this project is very lightweight and requires building docker images,
-   we are NOT currently run it using docker to avoid "Docker in Docker" (DinD) 
-   for the moment. This will likely change in the future as the need arises. 
-1. Install python 3.5
-1. Install [docker toolbox](https://www.docker.com/toolbox)
```
Comment (JulianGindi) [inline README.md:13]: I think we still need some information about getting Docker up-and-running

Comment (CorneliusIV) [inline README.md:13]: Should I leave what we had before with just installing the docker toolbox? I feel like there is more to it.

Comment (JulianGindi) [inline README.md:13]: I think we can just link to the official "getting started" documentation. Brings you from 0 to hello-world in docker. https://docs.docker.com/engine/installation/mac/

**Comment by JulianGindi** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (CorneliusIV) [inline README.md:13]: Done

**Pull request merged**